### DISCUSSION TITLE
Current priorities: beginning of 2023

### DISCUSSION THREAD

**Discussion by krzema12** [typesafegithub/github-workflows-kt#633] [Polls]
I have my take on this, but I'd like to ask you - my customers - what you think :smiley:

**Comment by LeoColman** [comment]
Stabilizing the library enough for a 1.0.0 release would be the most critical for adoption IMO

**Reply by krzema12** [reply]
From you point of view and for your particular use case, is the experimental status problematic (because e. g. in your company, you're allowed to use only stable libs)? Or do you mean in general?

**Reply by LeoColman** [reply]
Currently I'm using the library in pet OSS projects, and being experimental is not at all bad in that scenario.

However, when trying to present it as an alternative to writing the ymls to my company, it got shot down as "It seems to have potential. Let's stick to the standard yml for now until they get the library prod-ready, then we rethink".

I think it was the experimental status that got it rejected. If the version was stable people here would at least read the features 😂

**Reply by krzema12** [reply]
Ok, that's very helpful, thanks 😄
